### PR TITLE
Add to conflicts "google/apiclient-services": ">=0.200.0" to solve API class not found error

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #

### CODE REVIEW

File: lib/Google/Cse.php [hunk 1]
```diff
@@ -138,9 +139,9 @@ public function load()
     public $query = '';
 
     /**
-     * @var array
+     * @var CustomSearchAPI\Search
```
Comment (blankse) [inline lib/Google/Cse.php]:
```suggestion
     * @var CustomSearchAPI\Search|null
```

File: lib/Google/Cse.php [hunk 2]
```diff
@@ -262,7 +263,7 @@ public function setRaw($raw)
     }
 
     /**
-     * @return array
+     * @return CustomSearchAPI\Search
```
Comment (blankse) [inline lib/Google/Cse.php]:
```suggestion
     * @return CustomSearchAPI\Search|null
```

File: lib/Google/Cse/Item.php [hunk 3]
```diff
@@ -15,12 +15,13 @@
...
 namespace Pimcore\Google\Cse;
 
+use Google\Service\CustomSearchAPI\Result;
 use Pimcore\Model;
 
 class Item
 {
     /**
-     * @var \Google_Service_Customsearch_Result
+     * @var Result
```
Comment (blankse) [inline lib/Google/Cse/Item.php:24]:
```suggestion
     * @var Result|null
```

**Comment by blankse** [comment]
@dvesh3 The static analysis Test in 6.9 doen't test with lowest dependencies. I think we must add the namespace version to composer.json

**Comment by dvesh3** [comment]
@blankse old class names should be still supported with the installed version. please see https://github.com/googleapis/google-api-php-client-services/issues/383

**Comment by blankse** [comment]
@dvesh3 But you use now the new class names. So Pimcore require "google/apiclient-services": "^0.201.0". When you execute composer with --prefer-lowest you will get error in phpstan I think.

**Comment by dvesh3** [comment]
> @dvesh3 But you use now the new class names. So Pimcore require "google/apiclient-services": "^0.201.0". When you execute composer with --prefer-lowest you will get error in phpstan I think.

yes, i agree with you! but we decided not to go with this change at first place, as it will still break custom implementations. so our first preference is that it should be fixed on third-party repo otherwise we need to provide some BC layer along with this change.

**Comment by blankse** [comment]
@dvesh3 So you can set "google/apiclient-services": ">=0.200.0" in conflict and use the namespaces only in 10.x

**Pull request merged**